man/lx.1/lx.1.xml

```diff
@@ -86,6 +86,12 @@
 			and outputs a program which implements that analyser.
 			The specification language is documented by &lx.5;</para>
 
+		<para>Lexing is the process of categorising a stream of items
+			by their spellings. The output from this process is
+			a stream of tokens, each of a specific lexeme category,
+			which are most commonly input to a parser responsible
+			for asserting the order of these tokens is valid.</para>
+
```

Comment:
The one thing I feel is missing is that lexer output is often not just a token but a token and the specific identifier/spelling associated with it.

Also, this is maybe not the place for it, but I think an example would help to clarify this enormously.  There are a lot of terms here (eg: tokens, lexeme category) , and I'm not sure if they're explained elsewhere.  Something like:

> As an example, a lexer for a C-like language would take the input
> `direction = "north";` and output the tokens 
> `IDENTIFIER EQUALS STRING_LITERAL SEMICOLON`, 
> where `IDENTIFIER` and `STRING_LITERAL` would have the associated spellings `direction` and `north`, respectively.